### PR TITLE
Add /contact page, use instead of email

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

* I noticed that the `email` in the footer didn't actually work, at least on my browser
* The expected /contact page didn't exist, most org sites want that.
* Add LinkedIn to footer socials
* Minor improvement to layout of snailmail contact

Rendered at https://20241224-contact-and-linkedi.ddev-com-front-end.pages.dev/

See especially the footer and /contact pages

### CODE REVIEW

File: src/const.ts [hunk 1]
```diff
@@ -11,17 +12,18 @@ export const GITHUB_ISSUES_URL = "https://github.com/ddev/ddev/issues"
 export const DOCUMENTATION_URL = "https://ddev.readthedocs.io/"
 export const DISCORD_URL = "/s/discord"
 export const EMAIL_URL = "mailto:support%40ddev.com"
```
Comment (bmartinez287) [inline src/const.ts:14]: This typo to be specific we need to replace %40 for @

**Comment by bmartinez287** [comment]
Screenshot 2024-12-25 at 9 12 54 PM
the email issue seems to carry the on both places.
It looks like it was a typo on that variable. i added a comment to that.

**Comment by bmartinez287** [comment]
other than that the contact page seems fine.

File: src/const.ts (hunk 1 above)
Comment (rfay) [reply src/const.ts:14]: This works in regular links, and the idea is to prevent web crawlers from capturing the email address.

Comment (bmartinez287) [reply src/const.ts:14]: oooh, that makes sense, I thought this was the original issue with the email on the footer.

**Review by bmartinez287 — APPROVED** [review]

**Review by stasadev — APPROVED** [review]
I added LinkedIn SVG and fixed this error in the dev console
image

Looks good to me!

**Comment by rfay** [comment]
You're awesome, thanks!

**Pull request merged**